src/blog-utils/StaticAddonCard/index.js

```diff
@@ -34,8 +34,8 @@ export const StaticAddonCardBase = ({
   }
 
   const summary = addon.summary ? addon.summary : addon.description;
-  const averageRating = addon.ratings ? addon.ratings.average : null;
-  const averageDailyUsers = addon.average_daily_users || null;
+  const averageRating = addon.ratings ? addon.ratings.average : 0;
+  const averageDailyUsers = addon.average_daily_users || 0;
```

Comment:
we're kind of testing for scenarios that can never happen here.  (but if we intentionally are, then what about when `addons.ratings` exists but `addons.ratings.average` doesn't?)

Reply:
Fixed.